lib/shipitron/server/docker/configure.rb

```diff
@@ -9,22 +10,48 @@ class Configure
         include ConsulKeys
 
         required :application
+        optional :registry
 
         before do
           configure_consul_client!
         end
 
         def call
-          docker_auth = begin
-                          key = fetch_key(key: "shipitron/#{application}/docker_auth")
-                          key = fetch_key!(key: 'shipitron/docker_auth') if key.nil?
-                          key
-                        end
-          auth_file = Pathname.new('/home/shipitron/.docker/config.json')
-          auth_file.parent.mkpath
-          auth_file.open('wb') do |file|
-            file.puts(docker_auth.to_s)
-            file.chmod(0600)
+          username = fetch_scoped_key('docker_user')
```

Comment:
Why did this change from `docker_auth` to `docker_user` & `docker_password`?

Reply:
Past-me did this and I'm struggling to remember why.

Reply:
Does this mean you'll need to have the new data structure in consul as well?

Reply:
Yes, it's already there.

Reply:
I think this had something to do with supporting credential helpers.